### ISSUE TITLE
Flash Card Compatibility issue?

### ISSUE THREAD

**Issue by Reaga** [barronwaffles/dwc_network_server_emulator#83]
So I'm wondering if anyone can explain this.

So I used the patch of this to patch each of the Three versions of Megaman Star Force (Dragon, Leo, Pegasus) on my R4 3DS. My idea had been to register each version as a FC in the other games (I later realized that for the FC features to work in this game, they both must be connected at the same time, so it wouldn't have worked anyway but that's not the point). So I created a FC for each game one by one after applying the patch to all of them.

However, what I noticed is that when I connect to the server with one, the last one I connected with loses its save file. Now, the flash cart doesn't go through the process of creating a new .SAV, and the .Sav is still noticeable when looking through the SD in Windows. However, when loading the Rom continue options are no longer available.

Basically:
Sign on to get Pegasus FC code.
Sign on to get Leo FC.
Sign on to get Dragon.
Find that Leo and Pegasus only allow New Game options, Dragon is still fine.
Re-enter to get Pegasus or Leo, and Dragon file stops working.

So, is this because they're basically different versions of the same game, or am I limited to one FC for my flashcart (and therefore one wifi game on my flashcart) at a time, period? Because I intend to move onto Starforce 2 eventually and don't want to connect it to this server if it'll remove my SF 1 save.

**Comment by AdmiralCurtiss** [comment]
I don't see how this relates to this service, this sounds like an issue with your flash card.

**Comment by Reaga** [comment]
Because it's only an issue with games that have connected to the server. Other games do not have this problem. And when not connecting between games, the files remained intact.
